src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx

```diff
@@ -1287,3 +1289,22 @@ medVtkImageInfo* vtkImageView3D::GetMedVtkImageInfo(int layer /*= 0*/) const
 
     return imageInfo;
 }
+
+
+void  vtkImageView3D::initImage(int pi_iLayer)
```

Comment:
is it possible to give it a clearer name? Something like "initializeTransferFunctions"

Reply:
ok